Importer.php

```diff
@@ -406,7 +417,13 @@ public function import($idSite, $viewId, Date $start, Date $end, Lock $lock, $se
             $this->importStatus->finishImportIfNothingLeft($idSite);
 
             unset($recordImporters);
-        } catch (DailyRateLimitReached $ex) {
+
+        } catch (DailyRateLimitReached  | CloudApiQuotaExceeded $ex) {
+            if($ex instanceof CloudApiQuotaExceeded){
+                ApiQuotaHelper::trackEvent('Internal Quota Exception Reached','Google_Analytics_Importer');
+            } else {
+                ApiQuotaHelper::trackEvent('Google Quota Exception Reached','Google_Analytics_Importer');
```

Comment:
fyi it'll be important to not call those methods static. Otherwise dependency injection won't work. It would always call the method on the GAImporter ApiQuotaHelerp. We should make the methods non-static and then call it via `->trackEvent(...`

Reply:
converted static methods to object methods @tsteur .